Dockerfile

```
@@ -0,0 +1,16 @@
+FROM ruby:3.2
+
+RUN apt-get update && apt-get install -y build-essential
+
+COPY . /intellij-rust.github.io
+WORKDIR /intellij-rust.github.io
+
+RUN gem install github-pages jekyll jekyll-feed
+RUN bundle add webrick
+RUN bundle update
+RUN bundle install
```

Every `RUN` statement in a Dockerfile creates a separate layer. Common practice is reducing the number of layers when they are not useful for caching.

```suggestion
RUN gem install github-pages jekyll jekyll-feed && bundle add webrick && bundle update && bundle install
```